client/src/components/UpcomingTripsCard/UpcomingTripsCard.stories.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import UpcomingTripsCard from './UpcomingTripsCard';
+
+storiesOf('UpcomingTripsCard', module)
+  .add("Default", () => <UpcomingTripsCard 
+    header="Upcoming Trips"
+    detailOne="3 Days, 2 Nights"
+    detailTwo="Resort Stay"
+    detailThree="Children Welcome"
+    detailFour="Guided Tours"
```

Comment:
If you find yourself adding letters or numbers to variable, parameter, or property names then you should use an array!